modules/plan/src/main/StripeClient.scala

```diff
@@ -14,11 +14,42 @@ final private class StripeClient(
   import StripeClient._
   import JsonHandlers._
 
-  def createCustomer(user: User, data: Checkout, plan: StripePlan): Fu[StripeCustomer] =
+  def sessionArgs(data: CreateStripeSession): List[(String, Any)] = {
+    List(
+      "payment_method_types[]" -> "card",
+      "success_url"            -> data.success_url,
+      "cancel_url"             -> data.cancel_url,
+    ) ++ (data.customer_id match {
+      case Some(cid) => List("customer" -> cid.value)
+      case None => List()
+    })
```

Comment:
alternative notation: `data.customer_id.map(cid => "customer" -> cid.value).toList`

Reply:
This is much nicer. Didn't realize option had a toList.